src/lib/web3-utils.js

```diff
@@ -0,0 +1,39 @@
+import env from '../environment'
+
+export function getUseWalletProviders() {
+  const providers = [{ id: 'injected' }, { id: 'frame' }]
+
+  if (env('FORTMATIC_API_KEY')) {
+    providers.push({
+      id: 'fortmatic',
+      useWalletConf: { apiKey: env('FORTMATIC_API_KEY') },
+    })
+  }
+
+  if (env('PORTIS_DAPP_ID')) {
+    providers.push({
+      id: 'portis',
+      useWalletConf: { dAppId: env('PORTIS_DAPP_ID') },
+    })
+  }
+
+  return providers
+}
+
+export function getUseWalletConnectors() {
+  return getUseWalletProviders().reduce((connectors, provider) => {
+    if (provider.useWalletConf) {
+      connectors[provider.id] = provider.useWalletConf
+    }
+    return connectors
+  }, {})
+}
+
+export function getNetworkName(chainId = env('CHAIN_ID')) {
```

Comment:
I find it kind of awkward to default the argument to the current environment, as a utility it might be better to be explicit about always requiring it.